src/bun.js/api/BunObject.zig

```diff
@@ -2563,16 +2597,157 @@ pub const Crypto = struct {
 
             const output_digest_slice: []u8 = &output_digest_buf;
 
+            switch (this.*) {
+                .evp => {},
+                .zig => |*inner| {
+                    inner.final(output_digest_slice);
+                    return encoding.encodeWithMaxSize(globalThis, BoringSSL.EVP_MAX_MD_SIZE, output_digest_slice[0..inner.digest_length]);
```

Comment:
`BoringSSL.EVP_MAX_MD_SIZE` should be a different value here right?

Reply:
`BoringSSL.EVP_MAX_MD_SIZE` is the max size of all digests, `output_digest_slice` is the size of the current digest